### PR TITLE
make sure build directory exists

### PR DESCRIPTION
**Pull request by eworm-de**

### CODE REVIEW

**Comment by eworm-de** [comment]
Oh, this does not work... Let me think about it.

**Comment by martanne** [comment]
The build directory should be created by the [dependency/sources](https://github.com/martanne/vis/blob/62ca03b94f09ddedc042bd5ac7bfa79215d7c380/GNUmakefile#L37) target, but apparently it does not work for you?

As for the `&&` no particular reason, feel free to clean it up.

File: GNUmakefile [hunk 1]
```diff
@@ -62,7 +65,7 @@ dependency/sources/ncurses-%: | dependency/sources
 	mv $@.part $@
 	[ -z $(LIBNCURSES_SHA1) ] || (echo '$(LIBNCURSES_SHA1)  $@' | sha1sum -c)
 
-dependency/build/libncurses-extract: dependency/sources/$(LIBNCURSES).tar.gz
+dependency/build/libncurses-extract: dependency/build /dependencysources/$(LIBNCURSES).tar.gz
```
Comment (martanne) [inline GNUmakefile]: typo, misplaced `/`

**Comment by eworm-de** [comment]
It works if I clone the git repository and build from a clean tree.

When I package this I do specify the source files in [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=vis-standalone) and place things where needed. Make does not download and dependency/sources target is not run.

Latest commit should be fine. ;)

**Comment by martanne** [comment]
Thanks, hopefully this concludes the build system saga. I think it is ready for a release now ...

**Comment by eworm-de** [comment]
Perhaps one last change... Just discovered that things break if wget is not installed in build chroot.
Want to commit b827a592?

**Comment by martanne** [comment]
The dependencies for the build directory should actually be order only. Does your stuff still work?

**Pull request merged**